src/components/Comment/Comment.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Comment.css';
+import PropTypes from 'prop-types';
+
+export const Comment = ({ comment: { name, email, body } }) => (
+
+  <div className="comment">
+    <h4 className="comment__name">
+      {name}
+    </h4>
+    <div className="comment__email">
+      {email}
+    </div>
+    <p className="comment__body">{body}</p>
+  </div>
+);
+
+export const commentData = {
```

Comment:
Дата - какой-то готовый контент с готовыми свойствами к использованию (подразумевает комментарий с юзером, текстом, тайтлом, временем). Увидев такое название, не сразу поймешь что там типизация.

Это лучше назвать по содержанию, чтобы сразу 
```suggestion
export const commentShape = {
```
```suggestion
export const commentTypes = {
```